app/src/mixins/search-mixin.ts

```diff
@@ -137,7 +144,10 @@ export class SearchMixin extends Mixins(CommonMixin) {
     // special case for conversion
     if (this.getEntityTypeCd && this.isConversion && type) {
       const nrRequestType = type as unknown as NrRequestTypeCodes
-      const entityType = ConversionTypes.find(conv => conv.value === nrRequestType)?.entity_type_cd || null
+      const entityType = ConversionTypes.find(conv =>
+        conv.value === nrRequestType && conv.origin_entity_type_cd === this.origin_entity_type_cd
+      )?.entity_type_cd || null
+
```

Comment:
Updated this logic to also check for the originEntityType since more than one ConversionType now maps back to the same NrRequestTypeCode (with the added entries for Continued-In business).